cloudevents/tests/test_pydantic_cloudevent.py

```diff
@@ -140,29 +159,44 @@ def test_http_cloudevent_mutates_equality(
     assert event1 != event2 and event3 != event1
 
 
-def test_cloudevent_missing_specversion():
+def test_cloudevent_missing_specversion(cloudevents_implementation):
+    errors = {
+        "v1": "value is not a valid enumeration member; permitted: '0.3', '1.0'",
+        "v2": "Input should be '0.3' or '1.0'",
+    }
```

Comment:
I'm not sure we should check the error string representation. I left it for consistency.